released/discovery_center/mission_3260/main.tf

```diff
@@ -4,26 +4,36 @@
 resource "random_uuid" "uuid" {}
 
 locals {
-  random_uuid               = random_uuid.uuid.result
-  project_subaccount_domain = "discoverycenter-tf-sap-ms-${local.random_uuid}"
-  project_subaccount_cf_org = substr(replace("${local.project_subaccount_domain}", "-", ""), 0, 32)
+  random_uuid       = random_uuid.uuid.result
+  subaccount_domain = lower(replace("mission-3260-${local.random_uuid}", "_", "-"))
+  subaccount_cf_org = substr(replace("${local.subaccount_domain}", "-", ""), 0, 32)
```

Comment:
We might want to provide a `cf_org_name` variable (default = "") for the user and only generate it if the user didn't set a value.

Reply:
Makes sense.